components/ProductsTable/ProductsTable.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="overflow-x-auto">
+    <table class="table">
+      <!-- head -->
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Stock</th>
+          <th>Price</th>
+          <th>Total Sales</th>
+        </tr>
+      </thead>
+      <tbody>
+        <!-- rows -->
+        <tr v-for="(user, index) in paginatedUsers" :key="index">
```

Comment:
Instead of `user` use more explicit name, ie `product`
Instead of `paginatedUsers` use more explicit name, ie `data`